application/src/main/java/org/togetherjava/tjbot/features/help/AutoPruneHelperRoutine.java

```diff
@@ -163,7 +163,9 @@ private void pruneMemberFromRole(Member member, Role role, ForumChannel helpForu
         guild.removeRoleFromMember(member, role)
             .flatMap(any -> member.getUser().openPrivateChannel())
             .flatMap(channel -> channel.sendMessage(dmMessage))
-            .queue();
+            .queue(null, failure -> logger
+                .debug("Failed sending a DM to user (%s) while pruning them from a helper role."
+                    .formatted(member.getId()), failure));
```

Comment:
use [logger formatter](https://github.com/apache/logging-log4j2#usage)

Reply:
It's now done! ✅